### ISSUE TITLE
UI Bootstrap tabs not working

### ISSUE THREAD

**Issue by pgcd** [ngbp/ngbp#220]
I have spent several hours and I cannot seem to make this work: 
in the example app, add the following to src/app/about/about.tpl.html

&lt;div>
  &lt;tabset>
    &lt;tab heading="first tab">content 1&lt;/tab>
    &lt;tab heading="second tab">content 2&lt;/tab>
  &lt;/tabset>
&lt;/div>

The expected result is to have the tabset converted to UL/LI etc; the actual result is that the tabset and tab tags are left alone and nothing happens.

**Comment by ToucheSir** [comment]
@pgcd This type of issue should be filed under [the ui-bootstrap repository](https://github.com/angular-ui/bootstrap). :)

**Comment by pgcd** [comment]
Well, not really: the problem is that the version of ui-bootstrap in ngbp 0.3 is so old that it uses a different directive - not the one in ui-bootstrap docs.

**Comment by joshdmiller** [comment]
You can easily update the version. It's all done through bower, so just update the version the normal way and then update the required files (if necessary) in `build.config.js`.

**Comment by pgcd** [comment]
Yes, that's what I ended up doing; still, I figured it'd be sensible to point it out as a way of helping other people who might have the same issue - feel free to close it =)

**Comment by ToucheSir** [comment]
Ah, my bad. It's unfortunate that bower doesn't have a mechanism for displaying "out of date" versions in `bower.json` before installing.
